google-cloud-firestore/src/main/java/com/google/cloud/firestore/BatchWriteResult.java

```diff
@@ -26,21 +26,18 @@
  */
 @InternalApi
 public final class BatchWriteResult {
-  private final DocumentReference documentReference;
+  private final String key;
```

Comment:
Also s/key/documentKey since this is not apparent from the classes context (or documentPath - see below).

Reply:
Thanks to IntelliJ, realized this field is no longer needed. Will remove in Node as well.